src/components/Editor/Footer.css

```diff
@@ -53,9 +53,20 @@
   fill: var(--theme-selection-color);
 }
 
-.source-footer > .commands > .action svg {
+.source-footer > .commands > .action > img.prettyPrint {
+  mask: url("images/prettyPrint.svg") no-repeat;
+  mask-size: 100%;
   height: 16px;
   width: 16px;
+  background: black;
```

Comment:
hmm, we should find the variable color that other buttons use. the variable helps us with dark theme :)